suite/tests/client-interface/drx-scattergather.dll.c

```diff
@@ -54,9 +54,41 @@ event_exit(void)
     dr_fprintf(STDERR, "event_exit\n");
 }
 
+static dr_emit_flags_t
+event_bb_app2app(void *drcontext, void *tag, instrlist_t *bb, bool for_trace,
+                 bool translating)
+{
+    instr_t *instr;
+    bool expanded;
+    bool scatter_gather_present = false;
+
+    for (instr = instrlist_first(bb); instr != NULL; instr = instr_get_next(instr)) {
```

Comment:
Given all the code violating the "rules" about no meta instrs, best to rule out metas.  Either change the iterator to app-only or check inside the loop.

Reply:
Ok! Done.